4.x: -Pjlink-image -Djlink.image.stripDebug=true fails with Unsupported class file major version 65

If you build a Helidon 4 application (in this case 4.0.2) using:

```
mvn clean install -Pjlink-image -Djlink.image.stripDebug=true
```
It fails with:

```
[ERROR] Failed to execute goal io.helidon.build-tools:helidon-maven-plugin:4.0.2:jlink-image (jlink-image) 
on project helidon-quickstart-se: Image generation error: Unsupported class file major version 65 -> [Help 1]
```

This is because the `linker` module uses an old version of asm (7.2).

--- a/maven-plugins/sitegen-maven-plugin/src/test/resources/basic2/_expected.ftl
+++ b/maven-plugins/sitegen-maven-plugin/src/test/resources/basic2/_expected.ftl
@@ -41,7 +41,7 @@ $ mvn
 <dl>
 <dt>asciidoctor-version</dt>
 <dd>
-2.0.16
+2.0.20
 </dd>
 <dt>safe-mode-name</dt>
 <dd>

--- a/pom.xml
+++ b/pom.xml
@@ -123,8 +123,8 @@
         -->
         <version.lib.apache-commons>3.10</version.lib.apache-commons>
         <version.lib.archetype-common>3.2.0</version.lib.archetype-common>
-        <version.lib.asciidoctorj>2.5.2</version.lib.asciidoctorj>
-        <version.lib.asm>7.2</version.lib.asm>
+        <version.lib.asciidoctorj>2.5.11</version.lib.asciidoctorj>
+        <version.lib.asm>9.5</version.lib.asm>
         <version.lib.checkstyle>10.12.6</version.lib.checkstyle>
         <version.lib.diffutils>2.2</version.lib.diffutils>
         <version.lib.doxia>1.11.1</version.lib.doxia>

